felix/bpf-gpl/tc.c

```diff
@@ -467,7 +492,24 @@ static CALI_BPF_INLINE void calico_tc_process_ct_lookup(struct cali_tc_ctx *ctx)
 		}
 	}
 
-	if (rt_addr_is_local_host(ctx->state->post_nat_ip_dst)) {
+	if (rt_addr_is_local_host(ctx->state->ip_src)) {
```

Comment:
Should this be limited to program types where the packet is leaving the host?  Don't want the outbound failsafe to kick in on a (spoofed) inbound packet.

Reply:
Ah OK, there's a CALI_F_TO_HEP condition inside this but it doesn't cover the `ctx->state->flags |= CALI_ST_SRC_IS_HOST;` line.